src/main/java/software/amazon/documentdb/jdbc/DocumentDbDatabaseMetaData.java

```diff
@@ -224,7 +224,7 @@ public ResultSet getTables(final String catalog, final String schemaPattern,
         if (isNullOrWhitespace(catalog)
                 && (types == null || types.length == 0 || Arrays.stream(types)
                         .anyMatch(s -> isNullOrWhitespace(s) || s.equals("TABLE")))) {
-            if (isNullOrWhitespace(schemaPattern)
+            if (schemaPattern == null
                     || properties.getDatabase().matches(convertPatternToRegex(schemaPattern))) {
                 addTablesForSchema(tableNamePattern, metaData);
```

Comment:
The tableNamePattern in the addTablesForSchema seems also need this change.

Reply:
On the oracle documentation, it says that 
`tableNamePattern - a table name pattern; must match the table name as it is stored in the database.` So I think that means tableNamePattern cannot be null. I will double check with our own JDBC documentation

Reply:
yea our own JDBC limitation doc does not say anything about tableNamePattern, so I'll remove the `isNullOrWhitespace(tableNamePattern)` check. I'll do the same regarding your other comment on `columnNamePattern` in `addColumnsForSchema`